client/app/worlds/worlds.controller.js

```diff
@@ -16,7 +18,10 @@ angular.module('farseerApp')
     }
 
     $scope.convertToLowerCaseNoSpaces = function(nameToConvert) {
-      return nameToConvert.replace(/ /g, '').toLowerCase();
+      if (nameToConvert !== null && nameToConvert !== undefined) {
```

Comment:
Technically, you can do a 'truthy' check with it and both null and
undefined return false. Its worth noting that these mean different things
in javascript (one is for explicit omission and the other for indicating a
property that was requested was not defined), so if only one is expected
you can narrow things down.

On Fri, Mar 4, 2016 at 2:01 PM, John Gartee <notifications@github.com>
wrote:

> In client/app/worlds/worlds.controller.js
> <https://github.com/FarseerTeam/Farseer/pull/38#discussion_r55073132>:
>
> > @@ -16,7 +18,10 @@ angular.module('farseerApp')
> >      }
> >
> >      $scope.convertToLowerCaseNoSpaces = function(nameToConvert) {
> > -      return nameToConvert.replace(/ /g, '').toLowerCase();
> > +      if (nameToConvert !== null && nameToConvert !== undefined) {
>
> Is there a built-in function that does this...or could it be extracted
> into your own method isNullOrUndefined(...), returning a boolean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FarseerTeam/Farseer/pull/38/files#r55073132>.
>